### PR TITLE
Test/landing

### PR DESCRIPTION
**Pull request by noahlissner**
Hi, yall.

Created a custom method for cypress located in cypress/support/inViewport.js, also created a test for scroll-snap functionality.

Please review.

### CODE REVIEW

File: cypress/integration/landing.spec.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+describe("Landing page", () => {
+    it("test scroll-snap functionality", () => {
+        cy.visit("http://localhost:1234")
+
+        cy.get('.bg-1').scrollTo('0%', '60%')
+
```
Comment (dennislundgren) [inline cypress/integration/landing.spec.ts:6]: Add 

'cy.tick(1500);'

Mandatory scroll doesn't scroll in an immediate fashion.

**Review by elilyx10 — APPROVED** [review]
Looks good! I approve.

File: cypress/integration/landing.spec.ts (hunk 1 above)
Comment (dennislundgren) [reply cypress/integration/landing.spec.ts:6]: Fantastic!

**Review by dennislundgren — APPROVED** [review]
Looks good!

**Pull request merged**